agents/smtp.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+from .agent import ServerAgent
+
+
+class SMTPAgent(ServerAgent):
+
+    def __init__(self, logfile):
+        super(SMTPAgent, self).__init__(logfile)
+
+        self.port = 25
+
+    def service_healthy(self):
+        try:
+            output = subprocess.Popen(
+                ["ps", "aux"], stdout=subprocess.PIPE
+            ).communicate()[0]
+
+            return (
+                self.port_open()
+                and any(
+                    process in output
+                    for process in ["postfix", "exim", "sendmail"]
+                )
```

Comment:
Process names in some environments might appear with different case (e.g., `Postfix`). Use `.lower()` just in case:
```
output = output.lower()
any(proc in output for proc in ["postfix", "exim", "sendmail"])
```

Reply:
Fixed this.